docs/jwst/outlier_detection/outlier_detection_ifu.rst

```diff
@@ -1,34 +1,47 @@
+serves as the basic format for all processing performed by this step.
 The basis of the outlier detection flagging for IFU data  is to look for pixels on the detector
 that are regularly discrepant from their neighbors, with a sharper division than could be explained
-by the detector PSF. The algorithm flags pixels that are  outliers when compared to their neighbors for a set of
-input files contained in an association. The neighbor pixel differences are the neighbors in the spatial direction.
-For MIRI data neighbor differences are found to the left and right of every
-science pixel, while for NIRSpec data the neighbor differences are
-determined from pixels above and below every science pixel. The difference between the MIRI MRS and NIRPSpec algorithm for finding the
-spatial pixel differences is due to the opposite dispersion directions between the two instruments. 
-The  pixel differences for each input model 
-in the association are determined and stored in a stack of pixel differences. 
-For each pixel the minimum difference
-through this stack is determined and normalized. The normalization uses a local median of the difference array
-(set by the kernel size). A pixel is flagged as an outlier if  this normalized minimum difference
-is greater than the input threshold percentage.  Pixels that are found to be outliers are flaged in in the DQ array.
-
-
-
-.. automodapi:: jwst.outlier_detection.ifu
+by the detector PSF.
+This routine performs the following operations:
+
+#. Extracts parameter settings from input ModelContainer and merges them with any user-provided values.
+   See :ref:`outlier detection arguments <outlier_detection_step_args>` for the full list of parameters.
+
+#. Loops over cal files, computing nearest-neighbor differences for each pixel
+   in the along-dispersion direction.
+   For MIRI, with the dispersion axis along the y axis, the neighbors that are used to
+   to find the differences are to the left and right of each pixel being examined.
+   For NIRSpec, with the dispersion along the x axis, the neighbors that are used to
+   find the differences are above and below the pixel being examined.
+   The smaller of the two (left, right or up, down) differences is stored as the difference value for each
+   pixel. This avoids artifacts from bright edges.
+
+#. Compares the nearest-neighbor differences across science exposures to find the minimum
+   neighbor difference at each detector pixel.
+
+#. Determines a local spatial median of the minimum difference array using a median filter with a kernel size
+   set by the user according to the ``kernel_size`` parameter.
+
+#. Normalizes the minimum difference array by the local median.
+
+#. Selects outliers by flagging those normalized minimum values larger than the ``threshold_percent``
+   parameter.
+
+#. Updates input ImageModel DQ arrays with mask of detected outliers.
+
+
+After launch it was discovered that the bad pixels on the MIRI detectors vary with time.
```

Comment:
This paragraph should probably go at the top, before going into the detailed procedure.